templates/todoSingle.html

```diff
@@ -2,34 +2,52 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-<div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
-</div>
 
 <div class="row">
     <div class="col-md-12">
-        <h2>Items</h2>
+        <h2> {{items.name}} </h2>
+        <h7>Description: {{items.desc}}</h7>
```

Comment:
`<h7>` isn't a recognised tag. You should use `<span>`, for an unstyled inline element. Avoid inventing your own tags unless you have a good reason.

But if you want a line break between each item, then instead of
```
<span>your text<span>
<br>
```
You can just use `<div>your text</div>`

Sometimes more semantically, you could use paragraph tags `<p>` but they come with a margin by default, that you may or may not want.